lib/interceptor/cache.js

```diff
@@ -319,8 +319,9 @@ module.exports = (opts = {}) => {
         return dispatch(opts, handler)
       }
 
-      const reqCacheControl = opts.headers?.['cache-control']
-        ? parseCacheControlHeader(opts.headers['cache-control'])
+      const rawReqCacheControl = normaliseHeaders(opts)['cache-control']
```

Comment:
For cache hits, `normaliseHeaders()` will be called three times (here, and in `handleResult()`, and in `makeCacheKey()`).

Would it be preferable to do that once?  E.g. mutating or recreating `opts` here?

Reply:
Yes, that'd be better.